### PR TITLE
Add a screenshot to DevTools page

### PR DESCRIPTION
**Pull request by mit-mit**
Adding a screenshot to make it possible to visually get an impression of what this is.

### CODE REVIEW

File: src/tools/dart-devtools.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ for Dart and Flutter.
 These tools are distributed as part of the `dart` tool
 and interact with tools such as IDEs, `dart run`, and `webdev`.
 
+<img src="/tools/DevTools.png" width="400">
```
Comment (parlough) [inline src/tools/dart-devtools.md:11]: Seems a bit small for me. Also can we add some sort of alt text?

```suggestion
<img src="/tools/DevTools.png" width="640" alt="Screenshot of DevTools' Memory page">
```

Comment (MaryaBelanger) [reply src/tools/dart-devtools.md:11]: Agreed, very small

**Comment by mit-mit** [comment]
I think this looks OK now?
https://dart-dev--pr4893-mit-mit-devtools-ojm2t1yu.web.app/tools/dart-devtools

**Review by parlough — APPROVED** [review]
That does look better, thanks!

I resized the source image and optimized it as it was quite large.

**Pull request merged**